RELEASE_NOTES.rst

```diff
@@ -41,6 +41,7 @@ ARTIQ-4
   clocks dynamically (i.e. without device restart) is no longer supported.
 * ``set_dataset(..., save=True)`` has been renamed
   ``set_dataset(..., archive=True)``.
+* :func:`~artiq.tools.verbosity_args` renamed to :func:`~artiq.tools.add_common_args`. New feature: adds an option to print the ARTIQ version.
```

Comment:
In general, we don't mention internal API changes in RELEASE_NOTES, only user-visible changes.

Reply:
This is important for developers building on ARTIQ. For example, @jordens uses `verbosity_args`: https://github.com/quartiq/ptb-drivers/blob/2fecc58297a4f9930171c82d75fcb2e27724f9c8/ptb/aqctl_ptb_shutter.py#L11-L12

If only public API changes should be included in RELEASE_NOTES, then that should be clarified in the pull request template.

Reply:
Sorry, this one is actually in the documentation. The release notes should be updated then.